data/migrations/20200109191108_initial.js

```diff
@@ -0,0 +1,36 @@
+
+exports.up = async function(knex) {
+  await knex.schema.createTable('transmission', (table) => {
+    table.increments('id')
+    table.string('type')
+        .unique()
+        .notNullable();    
+  });
+  await knex.schema.createTable('title', (table) => {
+    table.increments('id')
+    table.string('status')        
+        .notNullable()
+            
+  });
+  await knex.schema.createTable('cars', (table) => {
+    table.increments('id')
+    table.string('make')
+          .unique()
+          .notNullable();
+    table.string('model')
+          .notNullable();
+    table.string('vin')
+          .unique()
+          .notNullable();
+    table.integer('mileage');
+    table.integer('tansmission_id');
+    table.integer('title_id');
```

Comment:
if transmission and title were required foreign keys, we're currently missing the reference to make it a correct fk